src/ui/spring-pane/index.erb

```diff
@@ -85,7 +85,7 @@
             <div class="input-group-prepend">
               <span class="input-group-text text-monospace">a<sub>max</sub></span>
             </div>
-            <input id="max_a_<%= node_id %>" type="text" placeholder="<%= stats['max_acceleration'].round(digits_precision) %>" readonly class="form-control" style="text-align: right;">
+            <input id="max_a_<%= node_id %>" type="text" placeholder="<%= stats['max_acceleration']['value'].round(digits_precision) %>" readonly class="form-control" style="text-align: right;">
```

Comment:
This is probably already due in another merge request, but on trusssprings right now seems to fail. Or is it just my machine?

Reply:
Yes you're right, sorry... I deployed a different branch than `trusssprings` to the aws server so rob can try a couple of things, didn't think about you also using the server... Have you reinstalled docker already? :-)

Reply:
Yes you're right, sorry... I deployed a different branch than `trusssprings` to the aws server so rob can try a couple of things, didn't think about you also using the server... Have you reinstalled docker already? :-)

Reply:
I actually have it working locally, was just wondering about it ^^
Will take it out here then.